docs/source/services/timelock_service/product-changes.rst

```diff
@@ -0,0 +1,20 @@
+.. _product-changes:
+
+Developing a product to use the timelock service
+================================================
+ .. note::
+    We recommend only product instances deployed against Cassandra KVS to use the timelock server.
+
+All products deploying against the AtlasDB Timelock service should follow the following checklist.
+
+1. Ensure that the AtlasDB config contains :ref:`timelock-client-configuration`.
```

Comment:
Yep. I think it is mentioned in the linked client config page that you can't do that, but "contains" could be misinterpreted...